packages/react-native/src/evm/components/ConnectWalletDetails/ExportLocalWalletModal.tsx

```diff
@@ -14,11 +14,11 @@ import { ModalHeaderTextClose } from "../base/modal/ModalHeaderTextClose";
 import { useAddress, useWallet } from "@thirdweb-dev/react-core";
 import { PasswordInput } from "../PasswordInput";
 import * as FileSystem from "expo-file-system";
-import { LocalWallet } from "../../wallets/wallets/local-wallet";
-import { SmartWallet } from "@thirdweb-dev/wallets";
+import { LocalWallet, SmartWallet } from "@thirdweb-dev/wallets";
```

Comment:
Please, use the React Native version of LocalWallet, not the `wallets` package version

Reply:
Done